src/client/src/routes/MainRoute.tsx

```diff
@@ -20,6 +21,7 @@ const APPLICATION_DISCONNECT = 15 * 60 * 1000
 
 const config = getEnvVars(process.env.REACT_APP_ENV!)
 const LOG_NAME = 'Application Service: '
+const warnLogger = logdown(`app:${LOG_NAME}`, { prefixColor: 'Tomato' })
```

Comment:
And, since we have no other logger to speak of, we have no need for the name prefix `warn`, so:
```js
const logger = logdown(`app:services`, { prefixColor: 'Tomato' })
```